manifest.php

```diff
@@ -32,7 +32,7 @@
     'label' => 'Generis Core',
     'description' => 'Core extension, provide the low level framework and an API to manage ontologies',
     'license' => 'GPL-2.0',
-    'version' => '13.4.0',
+    'version' => '13.4.2',
```

Comment:
Why 2 bugfix versions and not `13.4.1`?

Reply:
I've opened this one from another branch: https://github.com/oat-sa/generis/pull/836.

It contains some constant value updates which are not BC that's it should be done in 2 (actually 3 as there's another one in `tao-core`) steps.